galleries/examples/lines_bars_and_markers/multivariate_marker_plot.py

```diff
@@ -47,9 +47,13 @@
 plt.show()
 
 # %%
+<<<<<<< HEAD
```

Comment:
I'd rather this first commit not have the conflict markers introduced if we aren't squashing.